src/TextArea/TextArea.less

```diff
@@ -79,6 +82,10 @@
   color: @neutral_dark_gray;
   background: @neutral_silver;
 
+  .TextArea--label {
```

Comment:
We should avoid nesting in basically all cases. Any way we can clean this up throughout this less file?